examples/common/git_hub_api.py

```diff
@@ -0,0 +1,41 @@
+import json
```

Comment:
nit: why is this file named `git_hub_api` and not `github_api`?

Reply:
I try to break things up based on capitalizations:
```
GitHub -> git_hub
Sonarcloud -> sonarcloud
Datadog -> datadog
```